Dockerfile

```
@@ -9,14 +9,12 @@ RUN go mod download
 
 COPY . .
 
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /ping-server
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /hermetic
 
 FROM golang:1.20 AS run-stage
```

```suggestion
FROM gcr.io/distroless/base-debian11
```

We prefer distroless images.

Don't see any reason to use `AS run-stage` as it is implicit and not referenced anywhere.

---

As this is a change that is strictly unreleated to the code this PR changes, I will change the image we use in a separate PR. Sounds good?

---

Solved by https://github.com/nlnwa/hermetic/pull/4